extension/src/popup/views/UnlockAccount/styles.scss

```diff
@@ -10,7 +14,7 @@
   }
 
   &__import-account {
-    margin-top: 7.2rem;
+    margin-top: auto;
```

Comment:
til using margin auto with flex > `position: absolute`  🕺 

I'll use that for https://github.com/stellar/freighter/issues/408